src/app/child-dev-project/attendance/attendance-summary/attendance-summary.component.ts

```diff
@@ -18,8 +18,5 @@ export class AttendanceSummaryComponent {
       // start with most summative column, usually displayed right-most in table
       .reverse();
   }
-  get columns() {
-    return this._columns;
-  }
```

Comment:
I generally prefer not to use getters in the template as these are executed hundreds of times. This was the only time this getter was used so it can also be removed.

Reply:
oh, true. I forgot about the potential change detection issues with this